example/MyFirstDrawing/MyFirstDrawingUITests/BrushToolView/GroupEraseSizeViewUITests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GroupEraseSizeViewUITests.swift
+//  MyFirstDrawingUITests
+//
+//  Created by Stant Macmini n04 on 09/07/19.
+//  Copyright © 2019 Stant 02. All rights reserved.
+//
+
+import XCTest
+import StantDrawing
+@testable import MyFirstDrawing
+
+class GroupEraseSizeViewUITests: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+        
+        app.launch()
+        
+        let btnErase = app.buttons["eraserfullIdentifier"]
+        btnErase.tap()
+    }
+    
+    override func tearDown() {}
```

Comment:
Is it ok if you don't have any code here on tearDown? Specially for actions like app.launch(), does it have a impact on tests or it is ok to leave it like that?

Reply:
It's not necessary. I've removed